example.js

```diff
@@ -3,14 +3,19 @@ var Ora = require('./');
 
 var spinner = new Ora({
 	text: 'Loading unicorns',
-	spinner: process.argv[2]
+	spinner: process.argv[2],
+	split: true
 });
 
 spinner.start();
 
 setTimeout(() => {
-	spinner.color = 'yellow';
-	spinner.text = 'Loading rainbows';
+	spinner.setColor('yellow');
```

Comment:
You can use [`Object.defineProperty`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty) to do the same thing.

```js
Object.defineProperty(this, 'color', {
  set: function(color) {
    // do whatever you want to do when `spinner.color` is set.
  }
});
```